docs/application/tizen-studio/rt-ide/rt-overview.md

```diff
@@ -1,12 +1,23 @@
-# RT IDE
+# Tizen RT
 
-The Tizen Studio for RT is an IDE that provides a lightweight RTOS (real-time operating system)-based application development environment that helps you develop, build, flash, and debug applications in Ubuntu.
+Tizen has been commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The goal of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
 
-To develop your applications, use the set of tools provided by the Tizen Studio for RT:
+Tizen RT is an RTOS-based lightweight platform.
 
--   [Getting Started](getting-started/overview.md)
--   [SmartThings&trade; Extension](smartthings-ext/overview.md)
+* Hardware targets
+Cortex-M/R processors with MPU, less than 2 MB RAM, and less than 16 MB Flash.
 
-## Related Information
-* Dependencies
- - Ubuntu Only
+* Development environment
+	- Tizen RT Studio for SmartThings
+	Tizen RT Studio 2.0 (or later) is used as an IDE for developing IoT applications with the Tizen IoT Platform and ARTIK 053(s).
+
```

Comment:
Please rephrase to:

Tizen RT Studio for SmartThings: Tizen RT Studio version 2.0 (or later) is used as an IDE for developing IoT applications with the Tizen IoT Platform and ARTIK 053(s).